packages/navigator/src/browser/navigator-contribution.ts

```diff
@@ -340,6 +352,44 @@ export class FileNavigatorContribution extends AbstractViewContribution<FileNavi
             tooltip: 'Collapse All',
             priority: 1,
         });
+
+        // Register 'more actions' toolbar item (consists of multiple commands for the workspace root).
+        // Commands are available when there is a single-root workspace currently opened.
+        const navigatorRegisterItem = (item: Mutable<TabBarToolbarItem>) => {
+            const commandId = item.command;
+            const id = 'navigator.tabbar.toolbar.' + commandId;
+            const command = this.commandRegistry.getCommand(commandId);
+            this.commandRegistry.registerCommand({ id, iconClass: command && command.iconClass }, {
+                execute: (w, ...args) => w instanceof FileNavigatorWidget
+                    && this.commandRegistry.executeCommand(commandId, ...args),
+                isEnabled: (w, ...args) => w instanceof FileNavigatorWidget
+                    && !!this.workspaceService.workspace && !this.workspaceService.isMultiRootWorkspaceOpened
```

Comment:
I think we should allow registering the sam items for multi-root as well. They should be either executed on the current selection or on the first root if there is no.

Also Sven wants to add toggle auto reveal in a follow-up PR and it should be there always regardless to whether multi root workspace opened or not.

Reply:
Sounds good, I'll make the adjustments.